pytest.ini

```diff
@@ -1,6 +1,6 @@
 [pytest]
 DJANGO_SETTINGS_MODULE=djangogirls.settings
-addopts = --ff --nomigrations
+addopts = --ff --nomigrations -p no:warnings
```

Comment:
> I've also silenced warnings for pytest so looking through output isn't plagued with RemovedInDjango30Warning

Yes. But next up is to fix those. 😄 

We need to revert to `contrib.admin`, from django-suit, which is now unmaintained. Then update Django and Python, both to latest, and then we can stay there. 

I'd be inclined to leave the warnings on. We **want** to see them, so we know to fix them... 🤔

Reply:
@carltongibson Yeah I get that, but there was just sooooooo many of them 😂  I've reverted that. (I'll just put it back in myself when I'm testing.)

Ditching django-suit looks like a fair task thanks to those custom action buttons it seems to provide - I've not used it before I've been through the event creation process here.

An alternative that is maintained that I use is [django-admin-interface](https://pypi.org/project/django-admin-interface/). Though I can appreciate if you're at the point of removing one admin app, you'd be hesitant to want another incase that also ceases development.

Reply:
Thanks @marksweb. 

Yes... Hmmm... I haven't seen that particular package before so we can discuss that, but we need to get off of django-suit, since it's not compatible with higher versions of Django. 

(My **preference** would then be to keep it as plain as possible to avoid the same after the next LTS, say... but let's chat on another issue about that.) 

Thanks again for picking this up. Really good to have you onboard!

Reply:
@carltongibson, I just looked at this and tested locally and everything passes. 

Thanks @marksweb for picking this up. 

Just one slight hiccup, I ran a build for this and it fails for versions 3.5 and 3.6 which we no longer support but were in our GitHub Actions setup during our recent migration from Python 3.4 to 3.7. I have since done another PR to remove them after which you can rebase this so it passes and we can merge. Sorry about that but we are trying to do baby steps in moving to later versions of Python.

Thanks again for snatching this up. You really made @carltongibson's and my work easier.

Reply:
@marksweb if you rebase after @amakarudze’s #616 we should be good.

Reply:
@carltongibson @amakarudze I've rebased main 👍